app/db_services/user_service.py

```diff
@@ -0,0 +1,48 @@
+from ..schema.schema import User
+from .. import mongo
+from bson import ObjectId
+from pymongo.errors import DuplicateKeyError
+
+def get_all_users():
+     
+    users = mongo.db.users.find()
+    user_list = [User(user['username'], user['email'],user['firstname'],user['lastname'],user['mobileno']).to_dict() for user in users]
+    return user_list
+
+def create_user(username, email,firstname,lastname,mobileno):
+    try:
+        print("in")
+        new_user = User(username=username,email=email,firstname=firstname,lastname=lastname,mobileno=mobileno)
+        new_user.save_to_db()
+        print(new_user)
+        return {"message": "user created successfully."}
+    except Exception as e:
+        return {"error": f"Error creating user: {'E11000 duplicate key error collection: userdata.users index: unique_email_index dup key'}"}
```

Comment:
Your error message should be generic, Not the error message thrown by the package